external/js/cothority/README.md

```diff
@@ -49,6 +49,37 @@ You must use the given script instead of `npm publish` because we need to publis
 the _dist_ folder instead. If you try to use the official command, you will get
 an error on purpose.
 
+### Message classes
+
+You can write a class that will be used when decoding protobuf messages by using
+this template:
+```javascript
+class MyMessage extends Message<MyMessage> {
+    /**
+     * @see README#Message classes
```

Comment:
I don't know if you expected this to work as a link, but it doesn't. (And it appears all over this PR too.)

Reply:
I know it doesn't work as a link but the goal was to centralize the comment to one place and not having this function laying there and there with no explanations. I wanted it to be lightweight but if you prefer me to use an actual link I'll change it.

Reply:
No, it's fine.